packages/runtime/src/agent/file_storage.ts

```diff
@@ -1,14 +1,25 @@
 export class FileStorageAgent {
-  // TODO: consider to apply zero-copy buffer
-  async simpleGet(url: URL | string): Promise<Uint8Array> {
-    return Deno.core.opAsync(
-      "op_file_storage_agent_simple_get",
-      url instanceof URL ? url.href : url
+  constructor(readonly configurationId?: string) {}
+
+  async acquire(configurationName: string): Promise<FileStorageAgent> {
+    const configurationId = await Deno.core.opAsync(
+      "op_file_storage_agent_acquire",
+      configurationName
     );
+
+    return new FileStorageAgent(configurationId);
+  }
+
+  // TODO: consider to apply zero-copy buffer
+  async simpleGet(path: string): Promise<Uint8Array> {
+    return Deno.core.opAsync("op_file_storage_agent_simple_get", {
+      path,
+      configurationId: this.configurationId,
```

Comment:
```suggestion
      configurationId: this.configurationId,
      path,
```

I think it will be better to put configurationId before path and other arguments.